istioctl/pkg/postinstall/k8s_util.go

```diff
@@ -0,0 +1,61 @@
+// Copyright Istio Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package postinstall
+
+import (
+	"fmt"
+
+	"istio.io/istio/pkg/config/schema"
+	appsv1 "k8s.io/api/apps/v1"
+)
+
+func verifyDeploymentStatus(deployment *appsv1.Deployment) error {
```

Comment:
i assume there's no change to any code which is moved in this PR - lmk if this is not the case.

Reply:
Little change in error message. I moved "Istio installation failed, incomplete or does not match" part to a function called `istioVerificationFailureError`.